### PR TITLE
Feature/disease model

### PR DESCRIPTION
**Pull request by KjellSwedin**

### CODE REVIEW

File: src/vivarium_csu_ltbi/components/disease.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
+#from vivarium.framework.state_machine import Trigger, Transition
```
Comment (halfhorst) [inline src/vivarium_csu_ltbi/components/disease.py]: may as well remove this

Comment (KjellSwedin) [reply src/vivarium_csu_ltbi/components/disease.py]: Thanks, done.

File: src/vivarium_csu_ltbi/tools/builder.py [hunk 2]
```diff
@@ -138,6 +118,10 @@ def write_demographic_data(artifact, location):
     key = 'cause.all_causes.cause_specific_mortality_rate'
     write(artifact, key, load(key))
 
+    # TODO - placeholder
+    key = f'cause.{TUBERCULOSIS_AND_HIV}.cause_specific_mortality_rate'
+    write(artifact, key, data.csmr_300)
```
Comment (halfhorst) [inline src/vivarium_csu_ltbi/tools/builder.py]: Yongquan says this is "csmr_298 + csmr_297 (CSMR for the whole model!)"

File: src/vivarium_csu_ltbi/components/names.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+LTBI_POSITIVE_HIV = 'ltbi_positive_hiv'
+PROTECTED_TB_SUSCEPTIBLE_HIV = 'protected_tb_susceptible_hiv'
+PROTECTED_TB_POSITIVE_HIV = 'protected_tb_positive_hiv'
+SUSCEPTIBLE_TB_POSITIVE_HIV = 'susceptible_tb_positive_hiv'
+SUSCEPTIBLE_TB_SUSCEPTIBLE_HIV = 'susceptible_tb_susceptible_hiv'
+
+
+# transition names (from, to)
+ACTIVETB_SUSCEPTIBLE_HIV_TO_ACTIVETB_POSITIVE_HIV = 'activetb_susceptible_hiv_to_activetb_positive_hiv'
+ACTIVETB_SUSCEPTIBLE_HIV_TO_SUSCEPTIBLE_TB_SUSCEPTIBLE_HIV = \
```
Comment (halfhorst) [inline src/vivarium_csu_ltbi/components/names.py:18]: accidental line break here

File: src/vivarium_csu_ltbi/tools/builder.py (hunk 2 above)
Comment (halfhorst) [reply src/vivarium_csu_ltbi/tools/builder.py]: It's in the updated word doc with the tables of calculations

File: src/vivarium_csu_ltbi/components/names.py (hunk 3 above)
Comment (KjellSwedin) [reply src/vivarium_csu_ltbi/components/names.py:18]: It is actually too long given the super verbose names

File: src/vivarium_csu_ltbi/tools/builder.py (hunk 2 above)
Comment (KjellSwedin) [reply src/vivarium_csu_ltbi/tools/builder.py]: Good, change made

**Review by collijk — APPROVED** [review]

**Pull request merged**